### DISCUSSION TITLE
Busco equipo

### DISCUSSION THREAD

**Discussion by Israelote123** [Launch-X-Latam/MisionFrontEnd#507] [Team Posts]
Hola, buenos días a todos, busco personas para hacer equipo en el hackathon, soy de México
si quieren que hablemos pueden dejar su correo o mandarme un correo.
lx03559@innovaccion.mx

**Comment by EliasQuino** [comment]
Mi correo lx08662@innovaccion.mx

**Comment by CAMILOITT** [comment]
mi correo : lx11002@innovaccion.mx
